common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2495,6 +2495,11 @@
     ],
     "sqlState" : "23K01"
   },
+  "NO_MERGE_ACTION_ERROR" : {
```

Comment:
`NO_MERGE_ACTION_SPECIFIED`, I updated my review comment and you probably missed the update...

Reply:
Done